sparse_strips/vello_api/src/pixmap.rs

```diff
@@ -6,15 +6,17 @@
 use alloc::vec;
 use alloc::vec::Vec;
 
-/// A pixmap backed by u8.
+use crate::peniko::color::PremulRgba8;
+
+/// A pixmap of premultiplied RGBA8 values backed by [`u8`][core::u8].
 #[derive(Debug, Clone)]
 pub struct Pixmap {
     /// Width of the pixmap in pixels.  
-    pub width: u16,
+    width: u16,
     /// Height of the pixmap in pixels.
-    pub height: u16,
+    height: u16,
     /// Buffer of the pixmap in RGBA format.
-    pub buf: Vec<u8>,
+    buf: Vec<u8>,
```

Comment:
I'd love to make this `Vec<PremulRgba8>` :upside_down_face:

Reply:
I agree, I think we can do that; that's part of the motivation of this PR. I first wanted us to decide on the API direction more generally.